modules/cpmso-yaml-provider-spec-openstack.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-provider-spec-openstack_{context}"]
+= Sample OpenStack provider specification
+
+When you create a control plane machine set for an existing cluster, the provider specification must match the `providerSpec` configuration in the control plane machine custom resource (CR) that is created by the installation program.
+
+.Sample OpenStack `providerSpec` values
+[source,yaml]
+----
+providerSpec:
+  value:
+    apiVersion: machine.openshift.io/v1alpha1
+    cloudName: openstack
+      name: openstack-cloud-credentials <1>
+      namespace: openstack-machine-api
+    flavor: m1.xlarge <2>
+    image:
+    kind: OpenstackProviderSpec <3>
+    metadata:
+      creationTimestamp: null
+    network: 
+      filter:
+      subnets:
+               filter:
+                   name: nlpf7lgp-cc933-vj9pr-nodes
+                   tags: openshiftClusterID=nlpf7lgp-cc933-vj9pr
+      rootvolume:
+           disksize: 30
+           sourceUUID: nlpf7lgp-cc933-vj9pr-rhcos
+      securityGroups:
+            filter:
+            name: nlpf7lgp-cc933-vj9pr-master <4>
+      serverGroupName: nlpf7lgp-cc933-vj9pr-master
+      serverMetadata:
+          Name: nlpf7lgp-cc933-vj9pr-master
+          openshiftClusterID: nlpf7lgp-cc933-vj9pr
+      tags:
+          openshiftClusterID=nlpf7lgp-cc933-vj9pr
+      trunk: true
+      userDataSecret:
+           name: master-user-data
```

Comment:
@EmilienM Does something like this seem reasonable to you?

```suggestion
providerSpec:
  value:
    apiVersion: machine.openshift.io/v1alpha1
    cloudName: openstack
    cloudsSecret:
      name: openstack-cloud-credentials <1>
      namespace: openshift-machine-api
    flavor: m1.xlarge <2>
    image: ocp1-2g2xs-rhcos
    kind: OpenstackProviderSpec <3>
    metadata:
      creationTimestamp: null
    networks:
    - filter: {}
      subnets:
      - filter:
          name: ocp1-2g2xs-nodes
          tags: openshiftClusterID=ocp1-2g2xs
    securityGroups:
    - filter: {}
      name: ocp1-2g2xs-master <4>
    serverGroupName: ocp1-2g2xs-master
    serverMetadata:
      Name: ocp1-2g2xs-master
      openshiftClusterID: ocp1-2g2xs
    tags:
    - openshiftClusterID=ocp1-2g2xs
    trunk: true
    userDataSecret:
      name: master-user-data
```

Reply:
@EmilienM Let me know, if the example needs to be changed.

Reply:
looks good to me at a first glance.

Reply:
So should I change it?

Reply:
Done.